README.md

```diff
@@ -40,6 +40,7 @@ Workflow engine using [WDL](https://github.com/broadinstitute/wdl/blob/wdl2/SPEC
   * [GET /workflows/:version/:id/outputs/:call](#get-workflowsversionidoutputscall)
   * [GET /workflows/:version/:id/logs/:call](#get-workflowsversionidlogscall)
   * [GET /workflows/:version/:id/logs](#get-workflowsversionidlogs)
+  * [GET /workflows/:version/:id/call-metadata](#get-workflowsversionidcallmetadata)
```

Comment:
@geoffjentry that seems like a pretty trivial mod, just some more k/v pairs for data we already have
@scottfrazer thanks, will fix